R/wwinference.R

```diff
@@ -218,9 +231,69 @@ wwinference <- function(ww_data,
     }
   }
 
-  return(out)
+  structure(out, class = "wwinference_fit")
```

Comment:
need a class validator or similar

Reply:
Example?

Reply:
https://adv-r.hadley.nz/s3.html

13.3

Reply:
Adding a constructor. Right now, only checking for object types. Can do fancier checking in a next PR.